Explorer/Assets/DCL_RenderPass_GenerateSkyBox.cs

```diff
@@ -0,0 +1,187 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+public partial class DCL_RenderFeature_ProceduralSkyBox : ScriptableRendererFeature
+{
+    public class DCL_RenderPass_GenerateSkyBox : ScriptableRenderPass
+    {
+        // public static RTHandle k_CameraTarget
```

Comment:
Do we need to leave the unused skybox code?

Reply:
Will remove and clean up